File: problem/ordinal/6/main.rb

# main.rb: main file for solutions to the triple problem
#
# This is appended to the submitted Ruby program and tests the triple function.
#
# The submitted file processes the standard input and this processes the
# "in.txt" file.
#
#
# Copyright (C) 2020-2021  Rudy Matela
# All rights reserved.
#
#
# This file is part of the CSCX Online Judge.
#
# This is to be appended to the submitted file.

# standard input is already processed
# as the submitted programmed is run when imported
File.foreach("in.txt") do |line|
  x = line.to_i
  puts "#{ordinal(x)} #{x}#{suffix(x)}"
end

File: problem/ordinal/6/main.lua
-- main.lua: main file for "triple" solution
--
-- Copyright (C) 2020-2021  Rudy Matela
-- All rights reserved.
--
--
-- This file is part of the CScx Online Judge.
--
-- This is appended to the submitted Lua program and tests the triple function.
--
-- The submitted file processes the standard input and this processes the
-- "in.txt" file.
io.input("in.txt")
x = io.read("*n")
while x do
  print(ordinal(x) .. " " .. tostring(x) .. suffix(x))
  x = io.read("*n")
end
